pkgs/applications/editors/emacs/elisp-packages/manual-packages/holo-layer/default.nix

```
@@ -0,0 +1,79 @@
+{
+  # Basic
+  lib,
+  melpaBuild,
+  fetchFromGitHub,
+  # Python dependency
+  python3,
+  # Emacs dependencies
+  markdown-mode,
+  posframe,
+  # Updater
+  unstableGitUpdater,
+  ...
+}:
+
+let
+
+  python-pkgs =
```

```suggestion
  pythonPkgs =
```

https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax